doc/advanced_configuration.rst

```diff
@@ -13,51 +13,70 @@ Within your Sphinx ``conf.py`` file you need to add a configuration dictionary:
 .. code-block:: python
 
     sphinxgallery_conf = {
-        'root_dir'          : '../examples',              # path to your examples scripts
-        'examples_gallery'  : 'auto_examples'}            # path where to save gallery generated examples
+        'examples_dir' : '../examples',              # path to your examples scripts
+        'gallery_dir'  : 'auto_examples'}            # path where to save gallery generated examples
 
 
 Directory paths are relative to your ``conf.py`` location.
 
+
 Linking to external documentations
 ==================================
 
-If you want to hyperlink commands in your example scripts you can. Again within
-your Sphinx ``conf.py`` file you need to add a configuration dictionary:
+Sphinx-Gallery enables you to hyperlink commands in your example scripts to the
+matching location in their online documentation . Have a look at this in action
+in our example :ref:`example_plot_gallery_version.py`.
```

Comment:
Indeed I have a hard time explaining this. I'll try to put a clickable code snippet bellow to exemplify this.

The CSS for such links it is a matter of choice how to put them. The read the docs css removes any style to hyperlinks, so it shall be enforced on top of it. My local sphinx default theme, colours blue all links and makes the underline upon hover. I see nilearn is always underlined plus drastic colour change upon hover, scikit-learn is always underline and slight colour change upon hover. Would there be a preferred one? I also guess it needs to be address in the webpage CSS rather than in the gallery CSS, right?